# request fields
---
# response fields
bool costmaps_resetted